test/functional/cmdLineTests/criu/src/org/openj9/criu/TimeChangeTest.java

```diff
@@ -245,6 +251,35 @@ private void testGetLastRestoreTime() {
 		}
 	}
 
+	private void testGetCRIUProcessRestoreStartTime() {
+		long criuProcessRestoreStartTime = InternalCRIUSupport.getCRIUProcessRestoreStartTime();
+		if (criuProcessRestoreStartTime != -1) {
+			System.out.println("FAILED: InternalCRIUSupport.getCRIUProcessRestoreStartTime() - " + criuProcessRestoreStartTime
+					+ " is not -1 before restore");
+		}
+		CRIUSupport criu = CRIUTestUtils.prepareCheckPointJVM(CRIUTestUtils.imagePath);
+		Instant instant = Instant.now();
+		long beforeCheckpointTime = instant.getEpochSecond() * NANOSECONDS_PER_SECOND + instant.getNano();
+		CRIUTestUtils.checkPointJVMNoSetup(criu, CRIUTestUtils.imagePath, false);
+		criuProcessRestoreStartTime = InternalCRIUSupport.getCRIUProcessRestoreStartTime();
+		long lastRestoreTime = InternalCRIUSupport.getLastRestoreTime() * NANOSECONDS_PER_MILLISECOND;
```

Comment:
Why do `getCRIUProcessRestoreStartTime()` and `getLastRestoreTime()` return times in different units?

Reply:
It seems like the unit of the time returned by `getCRIUProcessRestoreStartTime()` (nanoseconds since epoch)  should be the same as the unit of the time returned by `getLastRestoreTime()` (milliseconds since epoch).

@tajila would it be incorrect to modify `getLastRestoreTime()` to return the start time of the restored java process using `j9sysinfo_get_process_start_time(j9sysinfo_get_pid(), &time)`? This would align both functions to return time in nanoseconds since epoch.

Reply:
`getLastRestoreTime()` should return when the JVM resumes control which Im not sure is the same as `j9sysinfo_get_process_start_time(j9sysinfo_get_pid(), &time)` I think the later represents the time that CRIU calls `clone()` on the restored process which is then when the JVM resumes from `criu_dump()`

To keep units the same you could simply remove the division that we do for `getLastRestoreTime()` since the time is originally in nanos.

Reply:
The question of units is moot if neither function is needed; see my earlier comment https://github.com/eclipse-openj9/openj9/pull/18680#discussion_r1456222511

> What is the point of those two methods anyway?

Reply:
I removed the division that is done for `getLastRestoreTime()` to modify its unit to nanoseconds since epoch, and renamed `lastRestoreTimeMillis` to `lastRestoreTimeInNanoseconds`.